EduNLP/ModelZoo/rnn/rnn.py

```diff
@@ -66,6 +82,19 @@ def __init__(self, rnn_type: str, vocab_size: int, embedding_dim: int, hidden_si
             load_net(model_params, self, allow_missing=True)
 
     def forward(self, seq_idx, seq_len):
+        """
+
+        Parameters
```

Comment:
Please check whether should put it inside or outside of __init__, because I see some info seems to be missed in doc.